packages/kit/src/lib/HookSystem.ts

```diff
@@ -176,24 +241,40 @@ export class HookSystem<
 		return hooks;
 	}
 
+	/**
+	 * Returns list of hooks for a given name
+	 */
+	hooksForName(name: string): RegisteredHook[] {
```

Comment:
Per our Notion discussion, should we rename "name" to "type"? I think "type" makes more sense, but I don't feel strongly about it.

If we make that change, we'll need to update any references of a hook "name" to a hook "type".

Reply:
Done 👌